packages/cozy-sharing/src/components/Recipient.jsx

```diff
@@ -167,7 +167,7 @@ export class Status extends Component {
   }
 }
 
-const StatusWithBreakpoints = translate()(withBreakpoints()(withClient(Status)))
+const StatusWithBreakpoints = withBreakpoints()(translate()(withClient(Status)))
```

Comment:
Nit: a lodash's `flowRight` can make it easier to read (and/or to add another HOC)

Reply:
Also nit but the change is… useless, no? 😄